Example Mods/NoGravity/NoGravity.cs

```diff
@@ -4,33 +4,40 @@
 using System.Text;
 using UnityEngine;
 using System.Collections;
-[Info("No Gravity",
```

Comment:
Is this information completely removed or have you moved it somewhere else?

Reply:
It has been removed from [VTOLMOD.cs](https://github.com/MarshMello0/VTOLVR-ModLoader/pull/6/files#diff-f7257f0d321693e121be582f30befb82), instead users create a info.xml which is used instead.

Reply:
Ah alright